vortex-expr/src/datafusion.rs

```diff
@@ -0,0 +1,63 @@
+#![cfg(feature = "datafusion")]
```

Comment:
does this actually work or does this only flag the one import statement?

should this go in the module declaration instead?

Reply:
The exclamation mark means it applies to the file

Reply:
nvm apparently it works